partiql-logical-planner/src/graph.rs

```diff
@@ -135,25 +152,41 @@ impl<'a> Normalize<'a> {
 
 impl GraphToLogical {
     fn normalize(&mut self, elements: Vec<MatchElement>) -> Result<Vec<PathPattern>, String> {
-        Normalize::new(&self.graph_id).normalize(elements)
+        Normalize::new(&self.id_generator).normalize(elements)
     }
 
     fn expand(&mut self, paths: Vec<PathPattern>) -> Result<Vec<PathPattern>, String> {
+        debug_assert!(!paths.is_empty());
         // TODO handle expansion as described in 6.3 of https://arxiv.org/pdf/2112.06217
         // TODO   this will enable alternation and quantifiers
-        Ok(paths)
+        // first, add disjuncts for alternation/union
+        // second, fix iterations for quantifiers
+        // third, clean-up; delete duplicated anonymous edges and merge
```

Comment:
Are these comments still relevant?

Reply:
Yes. These are TODOs for the upcoming work on quantifiers.

Reply:
ack